solver/llbsolver/ops/exec.go

```diff
@@ -69,8 +69,8 @@ func cloneExecOp(old *pb.ExecOp) pb.ExecOp {
 	}
 	n.Meta = &meta
 	n.Mounts = nil
-	for i := range n.Mounts {
-		m := *n.Mounts[i]
+	for i := range old.Mounts {
```

Comment:
^ same issue for `ExtraHosts` 5 lines above

Reply:
Is there an issue with `ExtraHosts`? It looks okay to me. The code iterates over `n.Meta` but it appends to `meta`.

Reply:
correct, looked it wrong